de.metas.handlingunits.base/src/main/java/de/metas/handlingunits/shipmentschedule/api/impl/HUShipmentScheduleBL.java

```diff
@@ -298,7 +298,7 @@ public void generateInOuts(final Properties ctx,
 	 * @return shipment which is still open for the shipment schedule (first) and it's shipper transportation
 	 */
```

Comment:
i didn't look closely, but i guess this javadoc is redundant..please remove it, or (if it has unique infos) please integrate it which the interface's javadoc

Reply:
I left the KeepInSync note in the implementation. The rest shall only be in intercafe.